### PR TITLE
feat/delete wallet and organization

### PR DESCRIPTION
**Pull request by pranalidhanavade**
### What 

Implemented delete functionality for following records.
- Verification records
- Issuance records
- Connection records
- Member organization from ecosystem
- Organization wallet 
- Organization

### Why

- To enable efficient management and removal of outdated or unnecessary data, ensuring data accuracy and system performance.

### CODE REVIEW

**Review by bhavanakarwade — CHANGES_REQUESTED** [review]

File: src/api/organization.ts [hunk 1]
```diff
@@ -451,4 +452,109 @@ export const updatePrimaryDid = async (orgId: string, payload: IUpdatePrimaryDid
...
+
+	try {
+		return await axiosDelete(axiosPayload);
+	} catch (error) {
+		const err = error as Error;
+		return err?.message;
+	}
+};
+
+export const deleteIssuanceRecords = async (
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/api/organization.ts]: Resolved

File: src/api/organization.ts [hunk 2]
```diff
@@ -451,4 +452,109 @@ export const updatePrimaryDid = async (orgId: string, payload: IUpdatePrimaryDid
...
+
+	try {
+		return await axiosDelete(axiosPayload);
+	} catch (error) {
+		const err = error as Error;
+		return err?.message;
+	}
+};
+
+export const deleteConnectionRecords = async (
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/api/organization.ts]: Resolved

File: src/components/organization/DeleteOrganization.tsx [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+  interface IEcosystemRole {
+    id: string;
+    name: string;
+    description: string;
+    createDateTime: string;
+    lastChangedDateTime: string;
+    deletedAt: string | null;
+  }
+
+  interface EcosystemOrgs {
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/components/organization/DeleteOrganization.tsx]: Resolved

File: src/components/organization/DeleteOrganization.tsx [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+      const { data } = response as AxiosResponse;
+      if (data?.statusCode === apiStatusCodes.API_STATUS_SUCCESS) {
+        const orgData = data?.data;
+        setOrganizationData(orgData);
+      } else {
+        setError(response as string);
+      }
+    } catch (error) {
+      console.error('An error occurred:', error);
+      setError('An unexpected error occurred');
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/components/organization/DeleteOrganization.tsx]: Resolved

**Review by bhavanakarwade — APPROVED** [review]
LG

**Review by tipusinghaw — APPROVED** [review]

**Pull request merged**